SRS.md

```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
 
 ### 3.2.3 Availability
 
-< The system shall provide a contractual agreement with an internet service provider for T3 access with 99.9999% availability. >
+ The system shall provide a contractual agreement with an internet service provider for T3 access with 99.9999% availability. 
 
 ### 3.2.4 Security
 
-< The system shall use secure sockets in all transactions that include any confidential customer information.
+The system shall use secure sockets in all transactions that include any confidential customer information.
 
-The system shall automatically log out all customers after a period of inactivity. >
+The system shall automatically log out all customers after a period of inactivity. 
 
 ### 3.2.5 Interfaces
 
-##
+#### 3.2.5.1 User Interfaces
+- The Online Virtual Phone System shall provide a user-friendly web interface that allows users to make, receive, and manage virtual phone calls.
+- It should enable users to manage their account settings, view call logs, and utilize additional features such as voicemail, call forwarding, and conferencing.
+- The UI shall be accessible via web browsers and be responsive to ensure usability across various devices like computers, tablets, and smartphones.
```

Comment:
I think we have not mentioned web browsers anywhere so I'd think desktop application.